### PR TITLE
fix remaining lint errors

### PR DESCRIPTION
**Pull request by felangel**

### CODE REVIEW

File: performance_tests/flutter_bloc/bloc_builder/test_driver/main.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file:close_sinks
```
Comment (tenhobi) [inline performance_tests/flutter_bloc/bloc_builder/test_driver/main.dart]: Nit: there should better be a space.

File: performance_tests/flutter_bloc/bloc_provider/test_driver/main.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file:close_sinks
```
Comment (tenhobi) [inline performance_tests/flutter_bloc/bloc_provider/test_driver/main.dart]: Nit: there should better be a space.

File: performance_tests/flutter_bloc/bloc_builder/test_driver/main.dart (hunk 1 above)
Comment (felangel) [inline performance_tests/flutter_bloc/bloc_builder/test_driver/main.dart]:
```suggestion
// ignore_for_file: close_sinks
```

File: performance_tests/flutter_bloc/bloc_provider/test_driver/main.dart (hunk 2 above)
Comment (felangel) [inline performance_tests/flutter_bloc/bloc_provider/test_driver/main.dart]:
```suggestion
// ignore_for_file: close_sinks
```

**Comment by felangel** [comment]
@tenhobi added the space, good catch 👍

**Pull request merged**